website/docs/docusaurus.config.js.md

```diff
@@ -287,3 +287,27 @@ module.exports = {
   ],
 };
 ```
+
+### `configureBabel`
+
+The function to customize Babel configuration.
+
+- Type `(isServer: boolean) => Object`
+
+Example:
+
+```js title="docusaurus.config.js"
+module.exports = {
+  // Added suppport for Flow.
+  configureBabel: (isServer) => ({
+    presets: [
+      '@babel/preset-flow',
+      // The following lines are necessary,
+      // unless you want to configure Babel from scratch.
```

Comment:
Is it possible to somehow add custom Babel plugins _without_ adding these lines? I feel this is a little verbose.

Reply:
There is a package [babel-merge](https://www.npmjs.com/package/babel-merge) that seems to do the job. However, it is not maintained by babel so I'm not sure whether we should use it, instead of relying on babel's preset merging mechanism that is guaranteed to work.

Reply:
agree that it could be added by default?

Reply:
based on usages/issues that seems pretty reliable, but I think we should build our own preset so that we don't need this, as discussed here: https://github.com/facebook/docusaurus/pull/2903#issuecomment-641313847